Makefile

```diff
@@ -20,7 +20,12 @@ cmd/device-rest-go:
 	$(GO) build $(GOFLAGS) -o $@ ./cmd
 
 test:
-	$(GO) test ./... -coverprofile=coverage.out
+	$(GO) test -coverprofile=coverage.out ./...
+	$(GO) vet ./...
+	gofmt -l .
```

Comment:
@jamesrgregg @lenny-intel @iain-anderson Would it be worthwhile to standardize Makefile commands across device services? I notice some isolate linting/formatting and others treat as part of `test` command.

Reply:
@tobiasmo1 , this is the standard that all services are moving to. @iain-anderson just added updated the other devices services... 

Would be nice if we all didn't have a duplicate copy of those scripts in /bin.